src/utils.py

```diff
@@ -82,23 +90,92 @@ def validate_request_data(input_json):
     return True, None
 
 
-def persist_repo_in_db(data):
-    """Store registered repository in the postgres database."""
-    req = OSIORegisteredRepos(
-            git_url=data['git_url'],
-            git_sha=data['git_sha'],
-            email_ids=data['email_ids'],
-            last_scanned_at=datetime.datetime.now()
+def _to_object_dict(data):
+        """Convert the object of type JobToken into a dictionary."""
+        return_dict = { OSIORegisteredRepos.git_url:data["git_url"],
+                        OSIORegisteredRepos.git_sha:data["git_sha"],
+                        OSIORegisteredRepos.email_ids:data["email_ids"],
+                        OSIORegisteredRepos.last_scanned_at:datetime.datetime.now()
+                    }
+        return return_dict
+
+
+class DatabaseIngestion():
+
```

Comment:
@GeetikaBatra please run `check-docstyle.sh` and `run-linter.sh` locally to ensure code sanity.

Reply:
sure, on it.